.gitignore

```diff
@@ -71,3 +71,7 @@ etc
 /test/e2e/report
 /test/e2e/results
 /test/e2e/state
+
+# Temporary files made by yarn when running canary releases
+.yarn
+.yarnrc
```

Comment:
Do these also ignore the .cjs files that are appearing bellow on the files changes? 👇